### DISCUSSION TITLE
Disparity between the binary sensor “person occupancy” in Home Assistant and the “alert” events that can be seen in frigate/review or in “topic: frigate/reviews”

### DISCUSSION THREAD

**Discussion by Islautopia** [blakeblackshear/frigate#15152] [Ask A Question] [answered]
I am impressed by the project, in general, and the beta-15 versions in particular. Congratulations, you are doing an awesome job. This is becoming the best NVR on the market.  
I've had 15-beta2 for a few days now and I'm really enjoying it. Everything works very nicely. However, on some occasions, I have seen that there is, in my understanding, a discrepancy in the trigger that I used in home assistant (person occupancy) and the alert type events in Frigate. The configuration is set so that only person detection generates alerts and the rest are detections and it works perfectly. However, very occasionally, I see in HASS  the Frigate camera entity "person count"  = 1 or the binary sensor person occupancy to “true” without seeing that alert in Frigate or receiving the corresponding message from MQTT. Is this normal behavior? Is there something I don't know about this or is it a bug?, I want to insist that 99% of the time, is consistent the HASS binary sensor with the Frigate alert.

**Comment by hawkeye217** [comment, marked as answer]
The person occupancy sensor has fewer restrictions to ensure that it responds quickly for automations, since it is often used for things like turning on lights. In some cases, it is triggered by false positives, which may not create an alert.

For something more filtered, you could set up a template sensor with the /events or /reviews MQTT topic.

**Reply by Islautopia** [reply]
Thx!

**Reply by hawkeye217** [reply]
Happy to help.
